charts/harvester-seeder/templates/rbac.yaml

```diff
@@ -0,0 +1,194 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ include "seeder.fullname" . }}-manager-role
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
+  - list
+  - update
+  - watch
+  - create
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - bmc.tinkerbell.org
+  resources:
+  - machines
+  - jobs
+  - tasks
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - bmc.tinkerbell.org
+  resources:
+  - machines/status
+  - jobs/status
+  - tasks/status
+  verbs:
+    - get
+    - create
+    - patch
+    - update
+- apiGroups:
+  - metal.harvesterhci.io
+  resources:
+  - addresspools
```

Comment:
we may consider to use `*` for the `metal.harvesterhci.io` group since they all belongs to the seeder repo